cmd/config/view_test.go

```diff
@@ -5,44 +5,85 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"strings"
```

Comment:
`goimports` suggested
```suggestion
	"testing"
	"strings"

	"github.com/stretchr/testify/assert"
```

Reply:
thx for your suggestion.